src/Dynamo.All.sln

```diff
@@ -812,6 +814,14 @@ Global
 		{9B4FDC96-E2F9-4B8F-894A-4294405D50E7}.Release|NET60_Linux.Build.0 = Release|NET60_Linux
 		{9B4FDC96-E2F9-4B8F-894A-4294405D50E7}.Release|NET60_Windows.ActiveCfg = Release|NET60_Windows
 		{9B4FDC96-E2F9-4B8F-894A-4294405D50E7}.Release|NET60_Windows.Build.0 = Release|NET60_Windows
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Debug|NET60_Linux.ActiveCfg = Debug|NET60_Linux
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Debug|NET60_Linux.Build.0 = Debug|NET60_Linux
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Debug|NET60_Windows.ActiveCfg = Debug|NET60_Windows
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Debug|NET60_Windows.Build.0 = Debug|NET60_Windows
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Release|NET60_Linux.ActiveCfg = Release|NET60_Linux
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Release|NET60_Linux.Build.0 = Release|NET60_Linux
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Release|NET60_Windows.ActiveCfg = Release|NET60_Windows
+		{7DD8077A-201E-4C56-96C5-3C901A51BDF3}.Release|NET60_Windows.Build.0 = Release|NET60_Windows
```

Comment:
We probably do not need to build this on the Linux config. I can’t tell if that is already done

Reply:
Sorry, what is already done?

Reply:
Should I remove the net60_linux config for this (and other wpf) assemblies?

Reply:
That is just cleanup. We can leave it for another time since there are other net48 configs all across the codebase to get rid of